var/spack/repos/builtin/packages/clingo/package.py

```diff
@@ -118,3 +120,6 @@ def cmake_args(self):
             args += ["-DCLINGO_BUILD_WITH_PYTHON=OFF"]
 
         return args
+
+    def win_setup_rpath(self):
+        self.add_internal_links(os.path.join(self.prefix.Lib, "site-packages"))
```

Comment:
Should this only happen when building with `+python`?

You could also just make sure to only reference this directory if it exists. Also I think it would be clearer if this used `self.spec['python'].platlib` (the Spack `python` package.py includes logic for locating `site-packages` which can account for special cases).

Reply:
I can set it to only occur when the spec specifies python, but I will note that without that variant set, this does become effectively a no-op. 

This extra rpath needs to point to the `Clingo` Package's `site-packages`, not `Python`'s, as that is where the clingo `.pyd` file (python extension module) that requires a link to `clingo.dll` lives, and where Spack attempts to load it from. Unless `self.spec["python"].platlib` returns the dependent packages site-libs?

Reply:
> I can set it to only occur when the spec specifies python, but I will note that without that variant set, this does become effectively a no-op.

That's fine: if the directory doesn't exist, then it should be.

> Unless self.spec["python"].platlib returns the dependent packages site-packages?

Yes - in more detail: `platlib` is relative. Packages which depend on `python` get a per-class setting of `python_platlib` (you can see an example of `llvm` using this).